include/envoy/server/admin.h

```diff
@@ -15,7 +16,14 @@
 namespace Envoy {
 namespace Server {
 
-class AdminFilter;
+class AdminStream {
+public:
+  virtual ~AdminStream() {}
+  virtual void setEndStreamOnComplete(const bool& end_stream) PURE;
+  virtual void addOnDestroyCallback(std::function<void()> cb) PURE;
+  virtual Http::StreamDecoderFilterCallbacks* getDecoderFilterCallbacks() PURE;
```

Comment:
`virtual const Http::StreamDecoderFilterCallbacks& getDecoderFilterCallbacks() const PURE;`

return callbacks by const ref, and make the method const.